infrastructure/src/main/java/org/corfudb/infrastructure/LocalMonitoringService.java

```diff
@@ -23,7 +23,7 @@
  * Created by zlokhandwala on 11/2/18.
  */
 @Slf4j
-class LocalMonitoringService implements MonitoringService {
+class LocalMonitoringService implements ManagementService {
```

Comment:
what was motivation for renaming it?

Reply:
Now with the addition of AutoCommitService, it's better to call ManagementService instead of MonitoringService.